### PR TITLE
added monitor to track update global index bots balance #LSC-56

### PR DESCRIPTION
**Pull request by swelf19**

### CODE REVIEW

File: collector/monitors/balance_monitor.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	if err != nil {
+		return fmt.Errorf("failed to get \"%s\" account balance: %w", m.BotAddress, err)
+	}
+	err = resp.GetPayload().Validate(nil)
+	if err != nil {
+		return fmt.Errorf("failed to validate response: %w", err)
+	}
+	coins := resp.GetPayload().Result
+	for _, coin := range coins {
+		if coin.Denom == "uusd" {
```
Comment (pr0n00gler) [inline collector/monitors/balance_monitor.go:65]: constant? We already have one - `UUSDDenom`

File: collector/monitors/balance_monitor.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+			amount, err := strconv.ParseFloat(coin.Amount, 64)
+			if err != nil {
+				// if for some reason we cannot parse uusd float amount, just leave previous value
+				return fmt.Errorf("failed to parse coins uusd amount: %w", err)
+			}
+			m.balanceUST.Set(amount / 1000000)
```
Comment (pr0n00gler) [inline collector/monitors/balance_monitor.go:71]: 1_000_000 and constant is clearer, I think.

**Review by pr0n00gler — COMMENTED** [review]
And just a suggestion, but maybe position `Operator Bot Balance' dashboard with 'Update Global Index' dashborads together, because these metrics are related?

File: collector/monitors/balance_monitor.go (hunk 1 above)
Comment (swelf19) [reply collector/monitors/balance_monitor.go]: done

File: collector/monitors/balance_monitor.go (hunk 2 above)
Comment (swelf19) [reply collector/monitors/balance_monitor.go]: done

**Comment by swelf19** [comment]
> And just a suggestion, but maybe position `Operator Bot Balance' dashboard with 'Update Global Index' dashborads together, because these metrics are related?

moved all bot related panels together

**Review by oopcode — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**